.travis.yml

```diff
@@ -91,160 +53,150 @@ matrix:
     fast_finish: true
 
     include:
-        # Try MacOS X.
+
+      # Try MacOS X.
         - os: osx
           stage: Cron tests
-          env: SETUP_CMD='test --remote-data=astropy'
-               CONDA_DEPENDENCIES="$CONDA_ALL_DEPENDENCIES clang"
-               PIP_DEPENDENCIES="scikit-image jplephem $ASDF_PIP_DEP"
+          env: PYTHON_VERSION=3.7
+               TOXENV="test-alldeps"
```

Comment:
I'd prefer underscores to keep things more readable, and am generally in favour of spelling things out if possible, so `TOX_ENV` or `TOX_ENVIRONMENT`

Reply:
`TOXENV` is tox thing, allowing to select the environment instead of `-e`.

Reply:
Yes since this has specific meaning I think we should keep it as-is.

Reply:
Like for `TOXPOSARGS`, maybe define an empty default on top and have a comment there about what it does.

Reply:
I've added a comment at the top